corehq/apps/hqadmin/views.py

```diff
@@ -458,6 +459,49 @@ def page_context(self):
         return context
 
 
+class VCMMigrationView(BaseAdminSectionView):
+    urlname = 'vcm_migration'
+    page_title = ugettext_lazy("Vellum Case Management Migration")
```

Comment:
I understand why, but I still feel a little weird about using the name "Vellum" in HQ copy. It's a code name used privately by the dev team. I guess putting it on an admin page is fine...